packages/terra-file-path/src/FilePath.jsx

```diff
@@ -16,9 +26,43 @@ const propTypes = {
 
 const defaultProps = {};
 
-const FilePath = () => (
-  <div>test</div>
-);
+const FilePath = props => {
+  const {
+    items,
+    intl,
+  } = props;
+
+  const allItemsExceptLast = items.slice(0, items.length - 1);
+  const lastItem = items[items.length - 1];
+
+  const generateChildren = () => allItemsExceptLast.map((item) => [
+    <CollapsibleMenuView.Hyperlink
+      key={item.key}
+      text={item.text}
+      href={item.href}
+      onClick={item.onClick}
+      onBlur={item.onBlur}
+      onFocus={item.onFocus}
+      onKeyDown={item.onKeyDown}
+      onKeyUp={item.onKeyUp}
+      onMouseDown={item.onMouseDown}
+      variant={item.variant}
+    />,
+    <CollapsibleMenuView.Divider useChevronRightIcon key={`divider-${item.key}`} />,
+  ]).flat();
```

Comment:
flat() has to be used here instead of a React fragment due to Collapsible Menu View not spreading out the children.

Reply:
Could you elaborate on this more.  This also sounds like it would be good as a comment in the code.

Reply:
Added a comment in the code itself. Here's what I added:
.flat() is used here instead of a react fragment on purpose. The children that get passed down to CollapsibleMenuView will not work with react fragments. They will be passed down without being spread. This results in pairs of hyperlinks and dividers being grouped together as one, breaking how width calculation is handled for resizing.